bazel/linux/templates/runner.template.sh

```diff
@@ -94,6 +94,10 @@ SINGLE=""
 TMPDIR="${TEST_TMPDIR:-$(mktemp -d)}"
 OUTPUT_DIR=${TEST_UNDECLARED_OUTPUTS_DIR:-$(mktemp -d)}
 
+# The program in the emulator should write its exit status to this file.
+EXIT_STATUS_FILE=$TMPDIR/exit_status_file
+KERNEL_OPTS+=("EXIT_STATUS_FILE=$EXIT_STATUS_FILE")
```

Comment:
if we pass an OUTPUT_DIR, I would just assume a specific path in OUTPUT_DIR.

Side note: we should have a README documenting the contract between VM and runner scripts, and scripts inside the VM somewhere. Happy to start writing one.

Reply:
Changed `TMPDIR` to `OUTPUT_DIR`